src/presentation/components/organisms/CreateJob/EditJobForm.tsx

```diff
@@ -0,0 +1,175 @@
+import axios from "axios";
+import { ChangeEvent, FC, FormEvent, useEffect, useState } from "react";
+import { createJobUrl, fetchSkillsUrl } from "../../../../constants/constants";
+import { ISkill } from "../../../../domain/Skill";
+import CreateButton from "../../atoms/CreateButton";
+import classes from "./EditJobForm.module.css";
+import { useAppSelector } from "../../../hooks/reduxHooks";
+import { TextField, Button } from "@mui/material";
+// import useForm, { FormInitialValues } from "../../../hooks/useForm";
+// import { current } from "@reduxjs/toolkit";
+
+// interface IProps {
+//   className: string;
+// }
+
+type Props = {
+  // initialFormData: FormInitialValues;
+  buttonText: string;
+  // onSubmitLogic: (title: string) => void;
+};
+
+const JobForm: FC<Props> = (props) => {
+  const [jobTitle, setJobTitle] = useState("");
+  const [jobCity, setJobCity] = useState("");
+  const [jobCompany, setJobCompany] = useState("");
+  const [jobPayment, setJobPayment] = useState("");
+  const [jobDesc, setJobDesc] = useState("");
```

Comment:
- jobTitle
- jobCity
- jobCompany
- jobPayment
- jobDesc

let's replace them with `useForm`

Reply:
jobPayment can be string.
backend will make it number.